solr/solrj/src/java/org/apache/solr/common/cloud/DefaultZkCredentialsProvider.java

```diff
@@ -18,10 +18,27 @@
 
 import java.util.ArrayList;
 import java.util.Collection;
+import org.apache.solr.common.cloud.acl.DefaultZkCredentialsInjector;
+import org.apache.solr.common.cloud.acl.ZkCredentialsInjector;
+import org.apache.solr.common.cloud.acl.ZkCredentialsProvider;
 
 public class DefaultZkCredentialsProvider implements ZkCredentialsProvider {
 
   private Collection<ZkCredentials> zkCredentials;
+  protected ZkCredentialsInjector zkCredentialsInjector;
```

Comment:
I think this is unused in this class? Can we push it down to where it would actually be needed?